service/src/main/java/crawlercommons/urlfrontier/service/rocksdb/RocksDBService.java

```diff
@@ -960,7 +960,7 @@ public URLItem next() {
                     final int pos2 = schedulingKey.indexOf('_', pos1 + 1);
                     final int pos3 = schedulingKey.indexOf('_', pos2 + 1);
 
-                    fromEpoch = Long.parseLong(schedulingKey.substring(pos2 + 1, pos3));
+                    fromEpoch = Long.parseLong(schedulingKey, pos2 + 1, pos3, 10);
```

Comment:
can you please explain this change?

Reply:
Small performance improvement by avoiding creating a new String with substring